__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const app = require("../app.js");
+const data = require("../db/data/test-data");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe.only("app", () => {
+  describe("GET /api/categories", () => {
+    test("status:200 and response message", () => {
+      return request(app).get("/api/categories").expect(200);
+    });
+    describe("GET /api/categories", () => {
+      test("returns an array of objects with a slug and desc", () => {
+        return request(app)
+          .get("/api/categories")
+          .expect(200)
+          .then((res) => {
+            expect(typeof res.body.categories[0]).toBe("object");
+            expect(res.body.categories[0].hasOwnProperty("description")).toBe(
+              true
+            );
+            expect(Array.isArray(res.body.categories)).toBe(true);
```

Comment:
These test assertions can be cleaned up a little, the typeof and Array.isArray is not directly needed as this is indirectly tested for in the expect for hasOwnProperty. 

However this testing is just asserting that the first element of categories has a property called description, but instead should be that every element in categories should have this property as well as a slug property.

There should also be an assertion that also checks that categories has atleast 1 element within in.